roles/openshift_cfme/templates/miq-pv-region.yaml.j2

```diff
@@ -8,6 +8,6 @@ spec:
   accessModes:
     - ReadWriteOnce
   nfs: 
-    path: /exports/miq-pv02
+    path: {{ openshift_cfme_nfs_directory}}/miq-pv02
```

Comment:
Nit: can you ensure the variable name has spaces on either side after/before the `{{`/`}}`'s for consistency please?

`path: {{ openshift.... }}`